script.js

```diff
@@ -10,34 +14,70 @@ function computerSelection() {
-        (playerSelection === choiceScissors && computersMove === choiceRock) ||
-        (playerSelection === choicePaper && computersMove === choiceScissors)) {
-        return `You lost! ${computersMove.toUpperCase()} beats ${playerSelection.toUpperCase()}.`;  
+        winnerMessage = `Human wins! ${playerSelection.toUpperCase()} beats ${computersMove.toUpperCase()} `;
+        document.querySelector(".p1-score").textContent = (playerOneScore++) + 1;
     } else if (playerSelection === computersMove) {
-        return "Tie game, Play again.";
+        winnerMessage = `Draw Game! ${playerSelection.toUpperCase()} vs ${computersMove.toUpperCase()} `;
+    } else {
+        winnerMessage = `Computer wins! ${computersMove.toUpperCase()} beats ${playerSelection.toUpperCase()} `;
+        document.querySelector(".p2-score").textContent = (playerTwoScore++) + 1;
+    } 
+
+    roundWinnerMessageElement.innerHTML = winnerMessage;
+    displayFinalMessage();
 }
+
+
+//targets button content to give clickable feature
+var allButtons = document.querySelectorAll(".content");
+allButtons.forEach(buttons => {
+    buttons.addEventListener("click", getPlayerChoice); 
+});
+
+function getPlayerChoice(){
+    let playerSelection = this.getAttribute("data-playerChoices");
+    playRound(playerSelection);
 }
 
-function game() {
-    var resultsArray = [];
-    for (let i = 0; i < 5; i++) {
-        var result = playRound();
-        resultsArray.push(result);
-        console.log(result);
-    }
-    console.log(resultsArray);
+// displays final round message who reached numberOfRounds first and targets button content to disables clickable feature
+function displayFinalMessage() {
+    let p1Score = playerOneScore;
+    let p2Score = playerTwoScore;
+    let finalWinnerMessageElement = document.querySelector(".final-winner");
+    let finalWinner = "";
+    let allButtons = document.querySelectorAll(".content");
+    
+    if (p1Score === numberOfRounds) {
+        allButtons.forEach(buttons => {
+            buttons.removeEventListener("click", getPlayerChoice); 
+        });
+        finalWinner = "Congratulations! You beat the computer!";
+        document.getElementById("refreshContainer").style.display = 'block';
+        document.getElementById("refreshContainer").style.display = 'flex';
+    } else if (p2Score === numberOfRounds) {
+        allButtons.forEach(buttons => {
+            buttons.removeEventListener("click", getPlayerChoice); 
+        });
+        finalWinner = "The computer has won! Resistance is futile!";
+        document.getElementById("refreshContainer").style.display = 'block';
+        document.getElementById("refreshContainer").style.display = 'flex';
+    } 
+
+    finalWinnerMessageElement.innerHTML = finalWinner;
 }
 
-game();
\ No newline at end of file
+let newGame = document.querySelector('.refresh');
+newGame.addEventListener('click', refreshPage);
+
+function refreshPage() {
+    window.location.reload(true);
```

Comment:
Ewwww - this is not a good user experience. 

It's bad practice to have to hard refresh. 

Instead of hard refreshing to start a new game - Don't remove the event listeners at the end of a game. Instead have the getPlayerChoice function check to see if the game is still going or not